packages/p2p-v2/src/components/PaymentMethodsHeader/PaymentMethodsHeader.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Text } from '@deriv-com/ui/dist/components/Text';
+import ArrowLeftBold from '../../public/ic-arrow-left-bold.svg';
+import './PaymentMethodsHeader.scss';
+
+type TPaymentMethodsHeaderProps = {
+    onGoBack?: () => void;
+    title?: string;
+};
+
+/**
+ * @component This component is used to display the header of the PaymentMethods screen
+ * @param onGoBack - The function to be called when the back button is clicked
+ * @param title - The title of the header
+ * @returns {JSX.Element}
+ * @example <PaymentMethodsHeader onGoBack={onGoBack} title={title} />
+ * **/
+const PaymentMethodsHeader = ({ onGoBack, title }: TPaymentMethodsHeaderProps) => {
+    return (
+        <div className='p2p-v2-payment-method-form__header'>
+            <div className='p2p-v2-payment-method-form__back-button'>
+                {onGoBack ? (
+                    <ArrowLeftBold
+                        onClick={() => {
+                            onGoBack?.();
+                        }}
+                    />
+                ) : null}
```

Comment:
Can we write it like this? and we dont need the optional chaining ? since its already checking for onGoBack
```suggestion
                {onGoBack && (
                    <ArrowLeftBold
                        onClick={() => {
                            onGoBack();
                        }}
                    />
                ) }
```